rust/tw_keypair/Cargo.toml

```diff
@@ -29,6 +29,8 @@ blake2 = "0.9"
 curve25519-dalek = "3"
 digest = "0.9.0"
 sha2 = "0.9"
+bitcoin = { path = "../../rust-bitcoin/bitcoin", features = [ "rand-std" ] }
```

Comment:
As discussed before, we shouldn't forget to avoid using `bitcoin` in `tw_keypair`